Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1590,8 +1590,11 @@ private void AddLinks()
                 midpointVisual.transform.parent = midpoint.transform;
                 midpointVisual.transform.localScale = new Vector3(invScale, invScale, invScale);
                 midpointVisual.transform.localPosition = Vector3.zero;
+                
+                Bounds bounds = new Bounds(midpointBounds.center * invScale, midpointBounds.size * invScale);
+                bounds.size = bounds.size.RotateAround(bounds.center, midpointVisual.transform.localRotation);
```

Comment:
Change lgtm, question around testing of this + preventing future regressions - it feels like in the old picture, the colliders were clearly not completely surrounding the actual handles themselves - it feels like maybe we could have some unit test/check that the collider's bounds contains the actual object bounds themselves.

FWIW I think that this is relatively low value test, I'm just trying to understand how we can programatically (i.e. in code) immortalize correct behavior.

I'm curious to hear @julenka's thoughts on this one

Reply:
> it feels like maybe we could have some unit test/check that the collider's bounds contains the actual object bounds themselves.

I agree that it may seem low value, but it should also be a fairly easy test to add. And this would have the benefit of once the test is added we can be fairly sure that the behavior will not regress in the future. So I would vote for adding a test. It also follows our principle "add a test for every bugfix".

If we cannot get a test in, then I'd recommend at least filing a bug, referencing this PR, and then describing what the test should do